packages/dotcom-ui-header/src/__stories__/story.tsx

```diff
@@ -29,7 +37,6 @@ storiesOf('FT / Header', module)
   .add('Default header with drawer', () => {
     const knobs = {
       showSubNavigation: toggleShowSubNav(),
-      showUserNavigation: toggleUserStateOptions(),
       showMegaNav: toggleShowMegaNav(),
```

Comment:
Whoops: Restore the user navigation knob for the "default header with drawer"  `dotcom-ui-header` storybook story
```suggestion
      showMegaNav: toggleShowMegaNav(),
      showUserNavigation: toggleUserStateOptions(),
```